bundles/CoreBundle/Resources/translations/en.json

```diff
@@ -942,5 +942,7 @@
   "domain_label_width": "Domain label width",
   "redirect_performance_warning": "Using this feature has very bad effects on the performance of the entire application, do you really want to continue?",
   "Uncategorized": "Uncategorized",
-  "methods": "Methods"
+  "methods": "Methods",
+  "export_configuration": "Export Configuration",
```

Comment:
maybe mention JSON here

something "export as json" and "import from json" just to avoid confusion with the other import

Reply:
Done, thanks for the suggestion